pkgs/by-name/ve/verapdf/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitHub,
+  maven,
+  makeWrapper,
+  stripJavaArchivesHook,
+  jre,
+}:
+maven.buildMavenPackage rec {
+  pname = "verapdf";
+  version = "1.26.2";
+
+  mvnParameters = "-pl '!installer' -Dverapdf.timestamp=1980-01-01T00:00:02Z -Dproject.build.outputTimestamp=1980-01-01T00:00:02Z";
+
+  src = fetchFromGitHub {
+    owner = "veraPDF";
+    repo = "veraPDF-apps";
+    rev = "v${version}";
+    hash = "sha256-bWj4dX1qRQ2zzfF9GfskvMnrNU9pKC738Zllx6JsFww=";
+  };
+
+  mvnHash = "sha256-kDuVNdvXvetsTeIQ3osRMRbLx3BJaEq6OVS3OgLVfHg=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    stripJavaArchivesHook
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share
+    install -Dm644 greenfield-apps/target/greenfield-apps-1.26.0.jar $out/share/verapdf.jar
```

Comment:
It seems that the version in `greenfield-apps/target/greenfield-apps-1.26.0.jar` always ends with `.0`, irrespective of the actual patch version. To simplify future updates, it would be better to replace this with something like `greenfield-apps/target/greenfield-apps-${lib.versions.majorMinor version}.0.jar`.